examples/stun/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	"github.com/pions/webrtc"
+)
+
+func main() {
+	peerConnection, err := webrtc.New(&webrtc.RTCConfiguration{
+		ICEServers: []webrtc.RTCICEServer{
+			{
+				URLs: []string{"stun:stun.l.google.com:19302"},
+			},
+		},
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	if err := peerConnection.CreateAnswer(); err != nil {
+		panic(err)
+	}
+
+	log.Printf("Looping...")
+	for {
```

Comment:
I don't think it makes a difference but I just do `select{}` I don't know if conceptually is better/worse!

Reply:
gotcha.